qemu/tests/qemu_disk_img.py

```diff
@@ -189,7 +187,7 @@ def check_backingfile(self):
                 msg = ("Expected backing file is null")
                 msg += " Actual backing file: %s" % backingfile
                 raise exceptions.TestFail(msg)
-            elif backingfile != self.base_image_filename:
+            elif os.path.basename(self.base_image_filename) not in backingfile:
```

Comment:
it looks differe with original logical, how to ensure no impact exists test cases ?